blueocean-personalization/npm-shrinkwrap.json

```diff
@@ -3,14 +3,21 @@
   "version": "0.0.2-unpublished",
   "dependencies": {
     "@jenkins-cd/blueocean-core-js": {
-      "version": "0.0.124-SNAPSHOT-jm-b",
-      "from": "@jenkins-cd/blueocean-core-js@0.0.124-SNAPSHOT-jm-b",
-      "resolved": "https://registry.npmjs.org/@jenkins-cd/blueocean-core-js/-/blueocean-core-js-0.0.124-SNAPSHOT-jm-b.tgz"
+      "version": "0.0.124",
+      "from": "@jenkins-cd/blueocean-core-js@0.0.124",
+      "resolved": "https://registry.npmjs.org/@jenkins-cd/blueocean-core-js/-/blueocean-core-js-0.0.124.tgz",
+      "dependencies": {
+        "@jenkins-cd/design-language": {
```

Comment:
Need to cleanInstall again to make this disappear. You will have to rerelease core-js since I released one today too sorry.